controllers/controllers.js

```diff
@@ -37,4 +37,25 @@ exports.getComments = (req, res, next) => {
         res.status(200).send({comments})
     })
     .catch((err) => {next(err)})
+}
+
+exports.postComment = (req, res, next) => {
+    const newComment = req.body;
+    const {article_id} = req.params;
+    addComment(newComment, article_id).then((comment) => {
+        res.status(201).send({comment})
+    })
+    .catch((err) => {next(err)})
+}
+
+exports.patchArticle = (req, res, next) => {
+    if(!req.body.hasOwnProperty('inc_votes')){
+        return res.status(400).send({message: 'bad request'})
+    }
```

Comment:
this logic should really be in the models as a promise reject.
a controller is just there to pass data through.

Reply:
removed, added error code into handler instead